### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by skeating**
## Description

Move everything to main; this was reviewed before it went to develop

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated all documentation necessary.
- [ ] I have checked spelling in (new) comments.

## Testing
- [X] Testing is done automatically and codecov shows test coverage
- [ ] This cannot be tested automatically

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -244,45 +236,45 @@ if(WITH_WALL)
...
+option (LIBLX_SHARED_VERSION
 "Include version information into the compiled libLX shared library." ON)
-mark_as_advanced(LIBSBML_SHARED_VERSION)
+mark_as_advanced(LIBLX_SHARED_VERSION)
 
 # Enable the generation of unit tests. If enabled, all test runners
 # will be created and can be run with "make test" or ctest.
 # This won't work in Visual Studio 2003, so we disable this option there.
 #
 if(NOT ${CMAKE_GENERATOR} MATCHES "Visual Studio 6" AND NOT ${CMAKE_GENERATOR} MATCHES "Visual Studio 7")
```
Comment (mattagape) [inline CMakeLists.txt:275]: Does it work with Visual Studio Code (assuming that is something different to Visual Studio)?

File: CMakeLists.txt [hunk 2]
```diff
@@ -754,7 +746,7 @@ if(WITH_ZLIB)
 "The chosen zlib library does not appear to be valid because it is
 missing certain required symbols. Please check that ${LIBZ_LIBRARY} is
 the correct zlib library. For details about the error, please see
-${LIBSBML_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log")
+${LIBLX_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeError.log")
```
Comment (mattagape) [inline CMakeLists.txt:749]: Is a `/` needed immediately after `${LIBLX_BINARY_DIR}`?

**Review by mattagape — CHANGES_REQUESTED** [review]
Just a couple of questions to answer, please.

File: CMakeLists.txt (hunk 1 above)
Comment (skeating) [reply CMakeLists.txt:275]: Yes

File: CMakeLists.txt (hunk 2 above)
Comment (skeating) [reply CMakeLists.txt:749]: Since the libsbml version does not I assume this does not :-)

**Review by mattagape — APPROVED** [review]
Thanks.
The `develop` branch builds ok, and `ctest` passes, so I think this is good to go!

**Pull request merged**